ui/checkboxradio.js

```diff
@@ -0,0 +1,298 @@
+/*!
+ * jQuery UI Checkboxradio @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright 2014 jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ *
+ * http://api.jqueryui.com/checkboxradio/
+ */
+( function( factory ) {
+	if ( typeof define === "function" && define.amd ) {
+
+		// AMD. Register as an anonymous module.
+		define( [
+			"jquery",
+			"./core",
+			"./widget"
+		], factory );
+	} else {
+
+		// Browser globals
+		factory( jQuery );
+	}
+}( function( $ ) {
+
+var formResetHandler = function() {
+		var form = $( this );
+		setTimeout( function() {
+			form.find( ".ui-checkboxradio" ).checkboxradio( "refresh" );
+		} );
+	},
+	escapeId = new RegExp( /([!"#$%&'()*+,./:;<=>?@[\]^`{|}~])/g ),
+	widgetCount = 0;
+
+$.widget( "ui.checkboxradio", {
+	version: "@VERSION",
+	options: {
+		disabled: null,
+		label: null,
+		icon: true,
+		classes: {
+			"ui-checkboxradio-label": "ui-corner-all",
+			"ui-checkboxradio-icon": "ui-corner-all"
+		}
+	},
+
+	defaultElement: "<input type='checkbox'>",
+
+	_getCreateOptions: function() {
+		var disabled,
+			that = this,
+			options = this._super() || {};
+
+		// We read the type here, because it makes more sense to throw a element type error first,
+		// rather then the error for lack of a label. Often if its the wrong type, it
+		// won't have a label ( eg: calling on a div, btn, etc )
+		this._readType();
+		this._readLabel();
```

Comment:
How about `_findLabel()`? Makes more sense, since it looks at the content of the label elsewhere.